rockcraft/commands/lifecycle.py

```diff
@@ -0,0 +1,101 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
This is duplicated in utils.py due to some import issues when generating the snap. I can delete this file